Sources/Core/AWSSDKIdentity/Sources/InternalAWSSTS/IdentityProvidingSTSClient.swift

```diff
@@ -49,8 +61,10 @@ package struct IdentityProvidingSTSClient: AWSSDKIdentity.IdentityProvidingSTSCl
                 "STSWebIdentityAWSCredentialIdentityResolver: Failed to retrieve credentials from STS with web identity token."
             )
         }
+        var properties = Smithy.Attributes()
+        properties.set(key: AWSSDKIdentity.AWSIdentityPropertyKeys.credentialFeatureIDs, value: credentialFeatureIDs + [AWSSDKIdentity.CredentialFeatureID.CREDENTIALS_STS_ASSUME_ROLE_WEB_ID.rawValue])
```

Comment:
Add the credential feature ID that marks successful completion of STS assumeRoleWithWebIdentity call.